src/edu/washington/escience/myria/MyriaConstants.java

```diff
@@ -352,7 +352,8 @@ private MyriaConstants() {}
   public static final int PYTHON_EXCEPTION = -3;
```

Comment:
Types and special lengths are different -- The pytoh java protocol is
1. send Type ( or Special Length in case of error, null or end of stream) 
2. send number of bytes to be read. ( not sent in case of special lengths)
3. send data/code.( not sent in case of special lengths)
while special lengths and Type are sent at the first step, i prefer to have them be separate enums for code maintainability for later.